src/expensesAPI/expenses-controller.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+// Imports:
+const fs = require('fs');
+const path = require('path');
+const { getUsersFromDB } = require('../usersAPI/users-controller');
+
+// Path:
```

Comment:
it's awesome that you deal how to work with file system, you can also try to initialize your server and send the needed data (stored in array) to the user via send() method
did you try to do it? or just worked with data in files?